src/main/kotlin/fi/metatavu/noheva/api/ExhibitionDevicesGroupsApiImpl.kt

```diff
@@ -89,7 +89,8 @@ class ExhibitionDevicesGroupsApiImpl : ExhibitionDeviceGroupsApi, AbstractApi()
                     sourceDeviceGroup = sourceDeviceGroup,
                     targetRoom = sourceDeviceGroup.room
                         ?: return createBadRequest("Source device group $sourceDeviceGroupId has no room"),
-                    creatorId = userId
+                    creatorId = userId,
+                    targetContentVersionExhibition = null
```

Comment:
You should not be able to change targetContentVersionExhibition, the target exhibition needs to be derived from the targetRoom (as targetExhibition)